src/main/java/org/terasology/climateConditions/ClimateConditionsSystem.java

```diff
@@ -56,28 +70,38 @@ public void addHumidityModifier(float order, ConditionModifier humidityModifier)
         humidityModifiers.put(order, humidityModifier);
     }
 
-    public void configureTemperature(int seaLevel, int maxLevel, float diversity, Function<Float, Float> function,
-                                     float minimumValue, float maximumValue) {
+    public void configureTemperature() {
+        configureTemperature(32, 10,-10, 10, world, biome);
```

Comment:
A really really minor nitpick...
```suggestion
        configureTemperature(32, 10, -10, 10, world, biome);
```

Reply:
Well, I'd actually like to think that we should all use the same auto-formatter and enforce that it is applied. I _think_ it is configured in the IntelliJ setup we ship with the engine automatically (like the license header and other things).